src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -601,14 +714,30 @@ private function getDefinitionSchema(bool $v3, string $resourceClass, ResourceMe
         }
 
         $options = isset($serializerContext[AbstractNormalizer::GROUPS]) ? ['serializer_groups' => $serializerContext[AbstractNormalizer::GROUPS]] : [];
+
+        $schemaFormatter = null;
+        if ($v3) {
+            try {
+                $schemaFormatter = $this->schemaFormatterFactory->getFormatter($mimeType);
+            } catch (\Exception $e) {
```

Comment:
The default formatter's `supports` method should just always return `true` instead of thowing. It will allow to remove this hack.